Tita/Controllers/MainViewController.swift

```diff
@@ -0,0 +1,235 @@
+//
+//  MainViewController.swift
+//  Tita
+//
+//  Created by 혜인 on 2022/01/10.
+//
+
+import UIKit
+import SnapKit
+import Then
+
+class MainViewController: UIViewController, UICollectionViewDelegate, UICollectionViewDataSource, UICollectionViewDelegateFlowLayout, UITableViewDelegate, UITableViewDataSource {
+    
+    //MARK: - List
```

Comment:
```suggestion
    //MARK: - Cell Data Lists
```
cell들의 데이터들을 담아주는 리스트이니 이렇게 표현해주는 게 더 좋을 것 같네요

Reply:
알겠습니다~~